### DISCUSSION TITLE
rtsp simple server on Win10?

### DISCUSSION THREAD

**Discussion by bertvanduivendijk** [bluenviron/mediamtx#597] [Questions]
Hi im new in this kind of stuff. I read that the server can work on win10. Where can i find the docs to install the server on win10?
Please help me, i really want to use it on my pc.

**Comment by aler9** [comment]
I think the README is very clear on the question: "Download and extract a precompiled binary from the release page." Download the windows binary, extract it and double-click it.

[a comment by an automated account is not shown]
